overturemaps/core.py

```
@@ -125,7 +129,7 @@ def _dataset_path(overture_type: str) -> str:
     # complete s3 path. Could be discovered by reading from the top-level s3
     # location but this allows to only read the files in the necessary partition.
     theme = type_theme_map[overture_type]
```

One thing we should do here is validate the release against a list of known acceptable ones - somewhat similar to how the overture_type converts to a theme, though this would just be a check. We probably want to avoid forming that URL with an arbitrary string. 

One thing that's been outstanding is dynamically generating that valid list but maybe just having a fixed list is fine for now.

---

@jwass Wouldn't it be okay to let it fail if the user enters an invalid one? That's on them to format correctly for now. The reason I think a fixed list isn't a good idea is because it then requires a PR to be merged and the library to be updated every time there is a new release which makes this library much less usable. In that scenario there will always be lag between the release coming out and the maintainers getting the new release merged in. This makes the library unsuitable for ETL jobs